src/components/inputs/localized-rich-text-input/required-value-error-message.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import ErrorMessage from '../../messages/error-message';
+import messages from './messages';
+
+const RequiredValueErrorMessage = () => (
```

Comment:
This component is exactly the same as the one in `src/components/inputs/localized-multiline-text-input/required-value-error-message.js`.

Perhaps we could have it shared internally to avoid the duplication?

Reply:
I would do this as a followup since it's in other components as well pre-rich-text.